cds/modules/deposit/api.py

```diff
@@ -299,22 +309,28 @@ def _process_files(self, record_id, data):
             snapshot = self.files.bucket.snapshot()
             self._fix_tags_refs_to_master(bucket=snapshot)
             # dump after fixing references
-            data["_files"] = self.files.dumps(bucket=snapshot.id)
-            snapshot.locked = False
+            self.files.bucket.locked = False
```

Comment:
this is after files modules upgrades, right?

Reply:
yes. The call to `self.files.dumps(bucket=snapshot.id)` passing a bucket is breaking functionality after upgrade of the files modules. I changed the logic to a more deterministic in terms of bucker locking and used the sync method